BE/sidedish11/src/main/java/dev/codesquad/java/sidedish11/dto/ItemDetail.java

```diff
@@ -0,0 +1,58 @@
+package dev.codesquad.java.sidedish11.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import dev.codesquad.java.sidedish11.entity.DetailSection;
+import dev.codesquad.java.sidedish11.entity.Item;
+import dev.codesquad.java.sidedish11.entity.ThumbImage;
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+public class ItemDetail {
+    @JsonIgnore
+    private String hash;
+    private String top_image;
```

Comment:
애노테이션 사용시 필드와 이전 행 사이에 한 줄의 공백 만들어 주시고요. 
자바 필드명에 언더스코어 사용은 자제해 주세요.

```suggestion

@JsonIgnore
private String hash;

@JsonProperty(name = "top_image")
private String topImage;